### PR TITLE
Filter local path from URL

### PR DESCRIPTION
**Pull request by szepeviktor**
This is my first pass on the problem.

Needs docs: https://github.com/kasparsd/minit/wiki/Minit-Action-and-Filter-Hooks

### CODE REVIEW

**Comment by szepeviktor** [comment]
An example code could be provided with @markoheijnen's solution from #60

**Comment by szepeviktor** [comment]
@kasparsd Please advise.

File: include/minit-assets.php [hunk 1]
```diff
@@ -230,6 +230,17 @@ protected function get_asset_relative_path( $handle ) {
 			return false;
 		}
 
+		$local_path = get_local_path_from_url( $item_url );
```
Comment (kasparsd) [inline include/minit-assets.php]: @szepeviktor We should call the method of this class, correct? `$this->get_local_path_from_url()`.

**Comment by kasparsd** [comment]
Looks good @szepeviktor -- only one suggestion.

File: include/minit-assets.php (hunk 1 above)
Comment (szepeviktor) [inline include/minit-assets.php]: Yes.

**Comment by szepeviktor** [comment]
Could you merge it?

**Comment by kasparsd** [comment]
Thank you @szepeviktor!

**Pull request merged**